path_config.go

```diff
@@ -94,6 +94,11 @@ func pathConfig(b *jwtAuthBackend) *framework.Path {
 					},
 				},
 			},
+			"pass_namespace_in_state": {
```

Comment:
Nit you can probably drop "pass" and have it be `namespace_in_state`.

Reply:
Yep, that's better. Changed in bbfa423bfae5143a280ceede8feb711ff5108f6b.